library/src/containers/Channels/NewOperations.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { OperationComponent } from './NewOperation';
+import { Toggle } from '../../components';
+
+import { bemClasses } from '../../helpers';
+import { useSpec } from '../../store';
+import { PayloadType } from '../../types';
+import { CONTAINER_LABELS, CHANNELS_TEXT } from '../../constants';
+
+export const OperationsComponent: React.FunctionComponent = () => {
+  const channels = useSpec().channels();
+
+  if (!Object.keys(channels).length) {
+    return null;
+  }
+
+  const className = CONTAINER_LABELS.CHANNELS;
+  const header = <h2>{CHANNELS_TEXT}</h2>;
+
+  const operationsList: React.ReactNodeArray = [];
+  Object.entries(channels).forEach(([channelName, channel]) => {
+    if (channel.hasPublish()) {
+      operationsList.push(
+        <li key={channelName}>
+          <OperationComponent
```

Comment:
Why `OperationComponent` and not `Operation`?

Reply:
Old nomenclature in all components (and in Kyma) :) But you're right, we need to switch to names without `Component` in the name.